pkgs/development/php-packages/pinba/default.nix

```diff
@@ -15,6 +15,10 @@ buildPecl rec {
     sha256 = "0wqcqq6sb51wiawa37hbd1h9dbvmyyndzdvz87xqji7lpr9vn8jy";
   };
 
+  # Fix GCC 14 build.
+  # from incompatible pointer type [-Wincompatible-pointer-types
+  env.NIX_CFLAGS_COMPILE = "-Wno-error=incompatible-pointer-types";
```

Comment:
I think it's not the first time we have to do that (see 5d0b417dcb20a24e1584a9e34d2f6e411d326e2a). I am wondering if we shouldn't add this by default in `buildPecl` builder. WDYT @piotrkwiecinski, @jtojnar ?

Reply:
It's a workaround. In general it should be fixed upstream but this extension hasn't been updated for 3-4 years.

Reply:
Fair enough, I'll validate it.